third_party/2and3/redis/client.pyi

```diff
@@ -350,3 +354,9 @@ class Script:
     sha: Any
     def __init__(self, registered_client, script) -> None: ...
     def __call__(self, keys=..., args=..., client=...): ...
+
+class Monitor:
+    def __init__(self, connection_pool) -> None: ...
+    def __enter__(self): ...
+    def __exit__(self, *args): ...
+    def listen(self) -> Iterable[Mapping[Text, Any]]: ...
```

Comment:
We generally use concrete types for return values (with exceptions):
```suggestion
    def listen(self) -> Iterable[Dict[Text, Any]]: ...
```

Reply:
Since the `next_command()` method is at least described by the docstring, I believe it should be included as well.